site/src/sphinx/server-cors.rst

```diff
@@ -0,0 +1,106 @@
+.. _server-cors:
+
+Configure CORS
```

Comment:
How about matching the tense of this subhead with the subheads below?
`Configuring CORS` or `Allow any...`

Reply:
Changed to `Configuring`! Thanks.